newm-server/build.gradle.kts

```diff
@@ -125,6 +126,8 @@ dependencies {
     testImplementation(Dependencies.TestContainers.JUINT)
     testImplementation(Dependencies.TestContainers.POSTGRESQL)
 
+    implementation("io.github.oshai:kotlin-logging:${kotlinLoggingVersion}")
+
```

Comment:
Follow the same pattern that we have to all dependencies. Create a `KotlinLogging` object at  `Dependencies.kt`. Something like:
```
    object KoltinLoggin {
        private const val VERSION = "6.0.9"
        const val ALL = "io.github.oshai:kotlin-logging:$VERSION"
    }

```

Reply:
Thanks for letting me know ^^